fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1408,7 +1408,9 @@ public static FHIRSearchContext parseQueryParameters(String compartmentName, Str
         }
 
         FHIRSearchContext context = parseQueryParameters(resourceType, queryParameters, lenient);
-        context.getSearchParameters().addAll(parameters);
+
+        // Add the inclusion criteria search parameters to the front of the search parameter list
+        context.getSearchParameters().addAll(0, parameters);
```

Comment:
why is parameters a list?  shouldn't it also be 0 or 1 comparment params?

Reply:
Doesn't need to be a list - fixed.